package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "1.0.0-beta.20",
+  "version": "1.0.0-beta.21",
```

Comment:
I think we should change this and make it something more like 0.21.0 to be more consistent with semantic versioning.